xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -764,28 +775,35 @@
       #set ($matchType = 'partial')
     #end
   #end
-  #set ($filterQuery = "#livetable_getFilterQuery(""${safe_tableAlias}.value"" $matchType)")
+  #set ($filterQuery = "#livetable_getFilterQuery(""${safe_tableAlias}.value"" $matchType false $multilistsize)")
   #set ($whereSql = "${whereSql} and $filterQuery.trim()")
-  #livetable_addFilterParam($filterValue $matchType $whereParams)
+  #livetable_addFilterParam($filterValue $matchType $whereParams $multilistsize)
 #end
 
 
-#macro (livetable_getFilterQuery $column $matchType $isList)
+#macro (livetable_getFilterQuery $column $matchType $isList $multilistsize)
   #if ($matchType == 'partial' || $matchType == 'prefix')
     upper($column) like upper(?)
   #elseif ($isList)
     ? in elements($column)
+  #elseif (($multilistsize > 1) && ($matchType == 'exact'))
```

Comment:
yes but the problem is "mvn xar:format" doesn't work and I didn't understand why (I don't know enough Maven).

I launch
```
cd /home/git-wiki/xwiki-platform/xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui
mvn xar:format
```
and I obtain:
```
INFO] --- xwiki-commons-tool-xar-plugin:8.2-SNAPSHOT:format (default-cli) @ xwiki-platform-livetable-ui ---
juin 10, 2016 2:07:26 PM org.sonatype.guice.bean.reflect.Logs$JULSink warn
AVERTISSEMENT: Error injecting: org.xwiki.tool.xar.FormatMojo
java.lang.TypeNotPresentException: Type org.xwiki.tool.xar.FormatMojo not present
....
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.622s
[INFO] Finished at: Fri Jun 10 14:07:26 CEST 2016
[INFO] Final Memory: 21M/232M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.xwiki.commons:xwiki-commons-tool-xar-plugin:8.2-SNAPSHOT:format (default-cli) on project xwiki-platform-livetable-ui: Execution default-cli of goal org.xwiki.commons:xwiki-commons-tool-xar-plugin:8.2-SNAPSHOT:format failed: Unable to load the mojo 'format' in the plugin 'org.xwiki.commons:xwiki-commons-tool-xar-plugin:8.2-SNAPSHOT' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: org/xwiki/tool/xar/FormatMojo : Unsupported major.minor version 52.0
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.xwiki.commons:xwiki-commons-tool-xar-plugin:8.2-SNAPSHOT
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/root/.m2/repository/org/xwiki/commons/xwiki-commons-tool-xar-plugin/8.2-SNAPSHOT/xwiki-commons-tool-xar-plugin-8.2-SNAPSHOT.jar
[ERROR] urls[1] = file:/root/.m2/repository/dom4j/dom4j/1.6.1/dom4j-1.6.1.jar

```